extensions/amp-story/1.0/amp-story.css

```diff
@@ -241,6 +241,15 @@ amp-story-page[distance="2"] {
   transform: translateY(100%) !important;
 }
 
+amp-story-page [text-background-color] {
+  border-radius: 3px;
+  line-height: 1.5em;
+  padding: 2px 4px;
+  text-indent: 0;
+  box-decoration-break: clone;
+  -webkit-box-decoration-break: clone;
```

Comment:
Here you have two options:

1. Adding some `!important` statements for all these CSS properties, so they override publisher styles. Logic would be that if a publisher wants to use custom CSS properties, they should use their own CSS instead of this new attribute
2. Or you could move this to `amp-story-user-overridable.css`, so publishers know they can easily override it with their own CSS

We tend to do option 1 usually, but every case is different

Reply:
Makes sense! I opted for the first option. PTAL!